src/internal/m365/collection/drive/url_cache.go

```diff
@@ -35,7 +35,7 @@ type itemProps struct {
 
 var _ getItemPropertyer = &urlCache{}
 
-// urlCache caches download URLs for drive items
+// urlCache caches download URLs for onedrive file items
```

Comment:
the original was just slightly more correct, since oneDrive is one of many applications built on the drive api.
```suggestion
// urlCache caches download URLs for drive file items
```